### ISSUE TITLE
[BUG] Chia.exe (not chia.exe) eating SSD write resource a lot

### ISSUE THREAD

**Issue by Satoshi-Yoda** [Chia-Network/chia-blockchain#4979]
**Describe the bug**
After a few days of plotting with windows GUI there is processes that consumed a lot of SSD write resource. Look at the attached screenshot: there are some chia.exe plotters processes, each wrote about 1.5 TB per plot, which is fine, and Chia.exe processes that wrote about 16 TB in total, which is questionable.

**To Reproduce**
Version 1.1.5.
_Appears_: GUI, plotting several plots in parallel for a few days. To be exact: about 5 days, because 1.1.5 was released only 5 days ago. I am plotting in 6 named queues, and sometimes (about once per day or two) add more plots to each queue.
_Does not appears_: With plotting in windows CLI.
_Does not appears_: When the GUI is running less than 1 day, Chia.exe wrote only about several GB.

**Expected behavior**
Chia.exe I/O write bytes to increase linearly through the days, and stay within some sane limits.

**Screenshots**
![chia-exe](https://user-images.githubusercontent.com/2902647/118284195-d5553980-b4d8-11eb-9686-304f5773f684.png)

- OS: Windows 10
- CPU: Ryzen 3600X, 32 GB RAM, temp plotting SSDs and destination HDDs are separated from the system SSD.

**Additionally**
- CPU time for Chia.exe about 38 hours also is questionable, but not so annoying. For the first day Chia.exe consumes only about 30-40 minutes of CPU time.
- Only setting that I changed from default was log level WARNING -> INFO.

**Comment by gromran** [comment]
> I/O Write Bytes - The number of bytes written in input/output operations generated by a process, including file, network, and device I/Os.

**Comment by fiveangle** [comment]
I do see ridiculous levels of spamming in the log at INFO level, but that metric you are looking at is not on-disk writes.

With default configs and remote harvester, I observe approx. a megabyte/minute of debug logs at INFO level (approx 4GB/day).

So much of the "INFO" level messages really need to be deprecated to DEBUG level, but in general I don't think this bug is valid.

**Comment by taHC81** [comment]
Ain't that just a parent process and sum of all child processes?

**Comment by Satoshi-Yoda** [comment]
@taHC81 It does not look that way... Parent/child processes are shown on another tab.

@fiveangle You are correct! That is not Disk writes. I looked at Windows profiler (aka Resource Monitor) and there is just very small activity from Chia.exe. And I also looked at my SSDs TBW, and also this TB is not there. So my first assumption was wrong.

But, I monitored GUI plotter closely 4 days, and gathered screenshots with 24h interval:
![leak](https://user-images.githubusercontent.com/2902647/118697439-f3dc6d00-b817-11eb-9180-f2ab6e6b92db.png)
CPU usage clearly increasing, nearly double every day, so something is probably "leaking".
